packages/expo/bundledNativeModules.json

```
@@ -105,5 +105,6 @@
   "unimodules-app-loader": "~3.1.0",
   "unimodules-image-loader-interface": "~6.1.0",
   "@shopify/react-native-skia": "0.1.136",
-  "@shopify/flash-list": "1.1.0"
+  "@shopify/flash-list": "1.1.0",
+  "@sentry/react-native": "~4.1.3"
```

```suggestion
  "@sentry/react-native": "^4.1.3"
```

it's probably ok to use `^` assuming they don't make any breaking changes in minor releases

---

This comment didn't age well :)